content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -242,24 +244,20 @@ status:
 ```
 
 The new Pod conditions must comply with Kubernetes [label key format](/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).
-Since the `kubectl patch` command still doesn't support patching object status,
-the new Pod conditions have to be injected through the `PATCH` action using
-one of the [KubeClient libraries](/docs/reference/using-api/client-libraries/).
+Since the `kubectl patch` command doesn't support patching object status,
+applications (or [Operators](/docs/concepts/extend-kubernetes/operator/)) that want to
+set custom Pod conditions must use the `PATCH` action. You can use a
+[Kubernetes client library](/docs/reference/using-api/client-libraries/) to write code
+that sets custom Pod conditions for Pod ready++.
 
-With the introduction of new Pod conditions, a Pod is evaluated to be ready **only**
+For a Pod that's using custom conditions, that Pod is evaluated to be ready **only**
```

Comment:
nit: `Ready` vs ready?
Possible edit: A Pod, with custom conditions set, is evaluated as `Ready` only when both ...